pyroll/core/transport/hookimpls/roll_pass_exit.py

```diff
@@ -0,0 +1,37 @@
+from ..roll_pass_exit import RollPassExit
+from ... import RollPass
+
+
+def check_if_previous_is_roll_pass(self):
+    if not isinstance(self.prev, RollPass):
+        raise ValueError("Unit can only follow a RollPass unit.")
+
+
+@RollPassExit.length
+def length(self: RollPassExit):
+    check_if_previous_is_roll_pass(self)
+    return self.prev.exit_point + self.prev.roll.nominal_radius
```

Comment:
Why the exit point there? Or should this be radius-exit_point?

Reply:
Exit from roll pass, same as high point to the point where the material leaves the "shadow" of the roll. Which is just the radius. What is right is that it should be radius - exit point to enable elastic material behaviour.